### ISSUE TITLE
Added support one channel for multiple program url addresses.

### ISSUE THREAD

**Issue by oncebeloved** [kodi-pvr/pvr.iptvsimple#319]
for example.
Channel 1,
http://abc.com/channel1.m3u8;http://def.cn/channel1.m3u8;http://ghi.cn/channel1.m3u8

It is the same program. If signal one is not good, it will automatically switch to signal two for testing.Just when I switch to the next channel.

**Comment by phunkyfish** [comment]
Is this meant to be a feature request?

**Comment by phunkyfish** [comment]
I don’t think this is possible, there is no measure of stream quality nor the ability to switch streams during playback.
